test/runtests.jl

```diff
@@ -338,29 +338,34 @@ function test_rs_w_fig()
     tf = Figure(; size=(1600, 1200))  # resolution in pixels
 
     # Indicate factor values that are in the top 50 percentile
-    # tac_om_50 = ADRIA.sensitivity.outcome_map(rs, mean_s_tac, x -> any(x .>= 0.5), foi; S=20)
-    # ADRIA.viz.outcome_map!(
-    #     tf[1, 1],
-    #     rs,
-    #     tac_om_50,
-    #     foi;
-    #     axis_opts=Dict(:title => "Regions which lead to Top 50th Percentile Outcomes", :ylabel => "TAC [m²]"),
-    # )
-
-    # # Indicate factor values that are in the top 30 percentile
-    # tac_om_70 = ADRIA.sensitivity.outcome_map(
-    #     rs, mean_s_tac, x -> any(x .>= 0.7), foi; S=20
-    # )
-    # ADRIA.viz.outcome_map!(
-    #     tf[2, 1],
-    #     rs,
-    #     tac_om_70,
-    #     foi;
-    #     axis_opts=Dict(
-    #         :title => "Regions which lead to Top 30th Percentile Outcomes",
-    #         :ylabel => "TAC [m²]",
-    #     ))
-    # save("outcome_map.png", tf)
+    tac_om_50 = ADRIA.sensitivity.outcome_map(
+        rs, mean_s_tac, x -> any(x .>= 0.5), foi; S=20
+    )
+    ADRIA.viz.outcome_map!(
+        tf[1, 1],
+        rs,
+        tac_om_50,
+        foi;
+        axis_opts=Dict(
+            :title => "Regions which lead to Top 50th Percentile Outcomes",
+            :ylabel => "TAC [m²]",
+        ),
+    )
+
+    # Indicate factor values that are in the top 30 percentile
+    tac_om_70 = ADRIA.sensitivity.outcome_map(
+        rs, mean_s_tac, x -> any(x .>= 0.7), foi; S=20
+    )
```

Comment:
Minor comment, but the variable should be `tac_om_30` to represent >= 30th percentile

Reply:
Yes, I know this was an oversight from the previous implementation